cli/command/service/create.go

```diff
@@ -68,6 +68,8 @@ func newCreateCommand(dockerCli command.Cli) *cobra.Command {
 	flags.SetAnnotation(flagSysCtl, "version", []string{"1.40"})
 	flags.Var(&opts.ulimits, flagUlimit, "Ulimit options")
 	flags.SetAnnotation(flagUlimit, "version", []string{"1.41"})
+	flags.Var(&opts.devices, flagDevice, "Devices to add")
+	flags.SetAnnotation(flagDevice, "version", []string{"1.42"})
```

Comment:
This version is almost assuredly wrong. No idea if this should be 1.41, 1.42, or 1.43. I wasn't easily able to figure out what the current version is.

Reply:
Currently, the master's API version is 1.43: https://github.com/moby/moby/blob/master/api/common.go#L6